### DISCUSSION TITLE
Plusieurs statuts de gestionnaire

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1615] [Idées]
**Besoin :**
Besoin exprimé par des référentes restauration collective travaillant pour 2 Départements différents : avoir plusieurs statuts de gestionnaire. Ex. : Super-administrateur et administrateur.

**Intérêt :**
Pour deux des référentes, l'intérêt serait de pouvoir enlever un gestionnaire et d'en ajouter un autre quand il y a un changement de prestataire privé à la gestion de la restauration d'un collège.
Pour les deux autres référentes, l'intérêt serait d'être les seules autorisées à renseigner et modifier les chiffres : leurs relations avec les établissements les laissent penser que certains intendants seraient capables de modifier dans leur dos les chiffres qu'elles renseigneraient.

@JenniferStephan, @alemangui, @hfroot, qu'en pensez-vous?

**Comment by hfroot** [comment]
Je serai plutôt pour l'ajout de plus de transparence de qui a fait quoi (lié a ta suggestion ici https://github.com/betagouv/ma-cantine/discussions/1614) parce que un granularité des autorisations pourrait être difficile à implémenter niveau technique et UX, en plus que ça pourrait produire plus de demandes d'aide (genre "pourquoi est-ce que je peux pas faire X")

**Reply by alemangui** [reply]
D'accord avec Helen. De plus, chaque cantine a une organisation très différente, donc trop compliqué d'envisager un modèle de droits qui convient à tout le monde.

**Reply by ghost** [reply]
OK, ça marche. Est-ce que ce besoin (#1614) vous semble en revanche envisageable?
